content/300-guides/300-upgrade-guides/200-upgrading-versions/050-upgrading-to-prisma-3/150-referential-actions.mdx

```diff
@@ -44,15 +48,15 @@ You can also decide to skip checking the defaults and run a migration to update
 
 ### Using Introspection
 
-If you [Introspect](../../../../concepts/components/introspection) your database, the referential actions configured at the database level will be reflected in your Prisma Schema. If you have been using Prisma Migrate or `prisma db push` to manage the database schema, these are likely to be the [<=2.25.0 default values](#the-2250-default-referential-actions).
+If you [Introspect](/concepts/components/introspection) your database, the referential actions configured at the database level will be reflected in your Prisma Schema. If you have been using Prisma Migrate or `prisma db push` to manage the database schema, these are likely to be the [<=2.25.0](#defaultvalues) default values.
```

Comment:
Intentional that "default values" is not part of the link text any more?

Reply:
If you read it without the link it makes sense. It just so happens to be the same words, thats all 😄

These are likely to be the <=2.25.0 default values

Reply:
Hm, but isn't the link pointing to `<=2.25.0 default values` and hence should be fully as the link text, not just the `<=2.25.0`?

Reply:
true, true, good shout